doc/nrf/known_issues.rst

```diff
@@ -986,6 +986,11 @@ DRGN-15817: Possible assert when used together with 802.15.4 protocols when runn
 
 .. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
+DRGN-15338: Extended scanner may generate reports containing truncated data from chained advertising PDUs
+  The scanner reports partial data from advertising PDUs when there is not enough storage space for the full report.
+
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```

Comment:
This does not apply to v1-6-0.

Reply:
This applies to the text above for DRGN-15338. Was fixed in https://github.com/nrfconnect/sdk-nrfxlib/commit/76469a054694708a5f79b5e0eefa52a7b3a8ff9a